lib/flutter_zalo_login.dart

```diff
@@ -55,7 +56,7 @@ class ZaloLogin {
     await channel.invokeMethod('getInfo');
     final Map<dynamic, dynamic> result = await channel.invokeMethod('getInfo');
```

Comment:
**Suggestion:** Avoid calling the same method twice. Store the result of channel.invokeMethod('getInfo') in a variable and use it.
```suggestion
    final Map<dynamic, dynamic> result = await channel.invokeMethod('getInfo');
```